pkg/internal/cluster/logs/logs.go

```diff
@@ -64,20 +64,30 @@ func Collect(nodes []nodes.Node, dir string) error {
 			"docker-info.txt",
 		),
 	}
-	// add a log collection method for each node
+
+	// collect /var/log for each node and plan collecting more logs
+	errs := []error{}
 	for _, n := range nodes {
 		node := n // https://golang.org/doc/faq#closures_and_goroutines
 		name := node.String()
 		// grab all logs under /var/log (pods and containers)
-		cmd := node.Command("tar", "--hard-dereference", "-C", "/var/log", "-chf", "-", ".")
+		cmd := node.Command(
+			"sh", "-c",
+			// Tar will exit 1 if a file changed during the archival.
+			// We don't care about this, so we're invoking it in a shell
+			// And masking out 1 as a return value.
+			// Fatal errors will return exit code 2.
+			// http://man7.org/linux/man-pages/man1/tar.1.html#RETURN_VALUE
+			`tar --hard-dereference -C /var/log -chf - . || (r=$?; [ $r -eq 1 ] || exit $r)`,
```

Comment:
I would really prefer to not invoke shell, but this is not doing much and there actually doesn't seem to be another good solution.

there is no flag that disables this behavior, and we want to preserve other exit codes:
- from tar that are not equal to 1
- any exit code from `docker exec ...`